cypress/e2e/models/migration/applicationinventory/archetype.ts

```diff
@@ -92,16 +92,14 @@ export class Archetype {
         inputText(archetype.description, description);
     }
 
-    protected selectStakeholders(stakeholders: Stakeholders[]): void {
-        stakeholders.forEach(function (stakeholder) {
-            selectFormItems(archetype.stakeholder, stakeholder);
-        });
+    protected selectStakeholders(stakeHolders: string) {
+        inputText(archetype.stakeholder, stakeHolders);
+        cy.get("button").contains(stakeHolders).click();
```

Comment:
More than one stakeholder can be selected . It can accept a string[]

```
stakeholders.forEach(function (stakeholder) {
 inputText(archetype.stakeholder, stakeholder);
        cy.get("button").contains(stakeholder).click();
```

Reply:
No, this accepts a string only and not string[] because it processes each stakeholder name individually. Same with Stakeholder groups.